### PR TITLE
Integration Tests: Show log in case of error

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/androidTest/java/com/owncloud/android/FileIT.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void testCreateNonExistingSubFolder() {
 
         SyncOperation syncOp = new CreateFolderOperation(path, true);
         RemoteOperationResult result = syncOp.execute(client, getStorageManager());
-        assertTrue(result.isSuccess());
+        assertTrue(result.getLogMessage(), result.isSuccess());
```
Comment (ezaquarii) [inline src/androidTest/java/com/owncloud/android/FileIT.java]: That is going to work ok.

How about `RemoteOperationResult.toString()` to provide full insight into the object state? The additional benefit is that debugger uses `toString()`, making any code stepping-through very pleasant experience. And you can get free debug output in many other situations.

That is - of course - assuming that `toString()` is not used for human-readable output.

Comment (tobiasKaminsky) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: Nice idea.
This then needs to be changed on library, but I'll change it here to "toString()".

Comment (ezaquarii) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: The trick is, that if you are using an interface, you'll need to visit all implementations.

Comment (tobiasKaminsky) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: https://github.com/nextcloud/android-library/pull/256

Comment (ezaquarii) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: I'm wondering if this https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#toString(java.lang.Object) can do the job for you quickly. Apache has something similar, I believe.

Comment (AndyScherzinger) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: nextcloud/android-library#256 has been merged and jitpack build has succeeded

Comment (tobiasKaminsky) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: Then approve and merge? :tada:

Comment (AndyScherzinger) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: Well I thought you change it to `assertTrue(result.toString(), result.isSuccess());`?

Comment (tobiasKaminsky) [reply src/androidTest/java/com/owncloud/android/FileIT.java]: Ha, right. I only changed it at one occurrence…

**Comment by tobiasKaminsky** [comment]
So…let us see what we get if the IT tests ever fail again…

**Pull request merged**